weaviate/collections/classes/config.py

```diff
@@ -1690,11 +1689,7 @@ def dynamic(
             See [the docs](https://weaviate.io/developers/weaviate/configuration/indexes#how-to-configure-hnsw) for a more detailed view!
         """  # noqa: D417 (missing argument descriptions in the docstring)
         return _VectorIndexConfigDynamicCreate(
-            distance=distance_metric,
-            threshold=threshold,
-            hnsw=hnsw,
-            flat=flat,
-            quantizer=quantizer,
+            distance=distance_metric, threshold=threshold, hnsw=hnsw, flat=flat, quantizer=None
```

Comment:
should we remove quantizer completely?

Reply:
Perhaps but there could be a day when we actually do want to specify the quantizer at this level. However, this would not be a breaking change if we add it then so yes let's remove now

Reply:
Ah wait, it was actually because of the inheritance hierarchy for `_VectorIndexConfigDynamicCreate` since it inherits from `_VectorIndexConfigCreate` in order to meet the expected type of `collections.create(vector_index_config=)`. So this `=None` is required here, unless a large refactor is undertaken. WDYT?